docs/docs/contribute/docs/blog.md

```diff
@@ -0,0 +1,205 @@
+    description: Senior Software Developer, Example, Inc.
+    avatar: https://avatars.githubusercontent.com/u/...
+    url: https://github.com/sampleuser
+```
+
+## Update spelling/expect.txt file with your ID
+
+The spell checker will flag your user ID as an unrecognized word.
+You can manually add this string to the
+`.github/actions/spelling/expect.txt` file
+as part of your PR,
+although the easiest way to handle this is to push your PR
+then resolve the error as discussed on the
+[Spell Checker](spell-check.md)
+page.
+
+## Blog source code
+
+Your blog should be developed using the standard Git
+flow documented in
+[Working with Git](../general/git/index.md).
+When you have created a local branch:
+
+* Create a .md file in the `docs/blog/posts` directory.
+  Give the file a meaningful name;
+  remember that many people from different organizations
+  may be contributing to this directory.
+
+    You do not need to modify the `mkdocs.yml` file for your blog.
+
+* If your blog has graphics, screen shots, YAML files, etc.
+  that will be included,
+  create an `assets` subdirectory for those files
+  in a subdirectory that has the same name
+  as the root name of your .md file.
+  For example, if your source file is named `myblog.md`
+  (which is not actually detailed enough to be a good file name),
+  you need to create a myblog/assets subdirectory.
+
+* Follow the instructions in
+  [Blog front matter](#blog-front-matter)
+  to provide the metadata required by the blog plugin.
+
+* The text that follows the `#` line until the first `##` line
+  is the introduction to your article
+  and also the abstract that is displayed on the "Blogs" landing page.
+
+Other coding notes for blogs:
+
+* Blog posts are considered part of the MkDocs NAV path.
+  This means that:
+
+    * Use the practices documented in
+      [External links and internal cross-references](code-docs.md/#external-links-and-internal-cross-references)
+      for your blog.
+    * You can use a local build to render your blog locally as you write.
+      See
+      [Build documentation locally](local-building.md)
+      for details.
+    * The `readthedocs.build` preview associated with your PR
+      contains the rendered version of your blog
+      so that you and your reviewers can see it.
+
+## Blog front matter
+
+The blog plugin requires some information to manage the blog.
+This is specified as part of the file's metadata.
+Here we document the fields that are required.
+Additional fields can be added; see the
+[blog plugin documentation](https://squidfunk.github.io/mkdocs-material/setup/setting-up-a-blog/#writing-your-first-post)
+for more information.
+
+```md
+---
+date: YYYY-MM-DD
+authors: [<GitHub-UserID>]
+description: >
+  <Brief description of this blog, all in one source line>
+categories:
+  - <cat-1>
+  - <cat-2>
+  - ...
+comments: true
+---
+
+# <blog title>
+```
+
+### Blog front matter fields
+
+* **date** -- Date when blog was most recently posted
```

Comment:
```suggestion
* **date** -- Date when blog was most recently posted.
  This can also be in the future. Then, the blog post will be hidden until the date is reached.
```